traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/forms/Form.java

```diff
@@ -121,6 +128,18 @@ public void reload(int sectionId, int fieldId) {
         }
     }
 
+    /**
+     * Rebinds all non header items in the form. This can result in a structural change in the form.
+     */
+    public void reloadAllNonHeaders() {
+        for (int i = 0; i < nodes.size(); i++) {
+            FormNode t = nodes.get(i);
+            if(!t.isHeader){
+            t.model = null;
+            getAdapter().notifyItemChanged(i);}
+        }
+    }
```

Comment:
Maybe it's better to rename this class form "Form" to "FormRecyclerView"? It is not obvious that this class is actually a view from it's name.

Reply:
actually i thought about it and im not gonna change it. because im gonna change the implementation in near future and i might not even use recycler and i think the implementation better be encapsulated so it can be changed to any other view inside. from outside it is a form and we dont care how its being done altough then ill need to hide recycler details which i will

Reply:
Ok, makes sence